src/globus_cli/parsing/shared_options/__init__.py

```diff
@@ -357,3 +357,17 @@ def no_local_server_option(f):
             "remote connection."
         ),
     )(f)
+
+
+def local_user_option(f: C) -> C:
+    """
+    Option for setting the mapped local user used across multiple Transfer commands
+    """
+    return click.option(
+        "--local-user",
+        help=(
+            "Optional value passed to identity mapping specifying which local user "
```

Comment:
This reads slightly differently than the source and destination endpoint's help text.

Should this reference "the host endpoint's identity mapping" or "the server's identity mapping"? If this looks colloquially correct then you can mark this comment resolved because I simply don't know, but wanted to call this out.

Reply:
Its a little confusing, which is why I left it vague. The mapping is set by admins at the storage-gateway level, but the value is passed to gridftp at the mapped collection level.